### PR TITLE
chore(experemental-features): Enabled activity center and communities by default

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5922 
### What does the PR do

Enabled activity center and communities by default

### Affected areas

Communities and Activity Center

### CODE REVIEW

**Comment by elina2015** [comment]
@borismelnik I need your help to rebuild the app - now there is no version for macOS.

**Comment by elina2015** [comment]
@borismelnik could you help with a build for macOS?

File: src/app/modules/main/profile_section/notifications/module.nim [hunk 1]
```diff
@@ -115,19 +113,14 @@ method onToggleSection*(self: Module, sectionType: SectionType) =
...
-        let ind = self.view.exemptionsModel().findIndexForItemId(cg.id)
-        if(ind != -1):
-          continue
-        let item = self.createItem(cg.id, cg.name, cg.images.thumbnail, cg.color, joinedTimestamp = 0, item.Type.Community)
-        self.view.exemptionsModel().addItem(item)
-  else:
-    for item in self.view.exemptionsModel().modelIterator():
-      if(item.itemType == Type.Community and self.controller.removeNotifSettingExemptions(item.id)):
-          self.view.exemptionsModel().removeItemById(item.id)
+  let channelGroups = self.controller.getChannelGroups()
```
Comment (saledjenic) [inline src/app/modules/main/profile_section/notifications/module.nim:116]: @borismelnik if we don't have a feature to enable/disable communities any more, then you're free to remove entire `onToggleSection` proc, and as well connection from the controller's `init` proc.

**Review by saledjenic — CHANGES_REQUESTED** [review]
@borismelnik I see that 2 out of 3 points are done here, do you plan to do the third point (one for Community History Archive Protocol) in a new PR?

File: src/app/modules/main/profile_section/notifications/module.nim (hunk 1 above)
Comment (borismelnik) [reply src/app/modules/main/profile_section/notifications/module.nim:116]: @saledjenic  ok

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**